server/graphql/v2/object/Host.js

```diff
@@ -1137,6 +1145,130 @@ export const GraphQLHost = new GraphQLObjectType({
           };
         },
       },
+      hostedAccounts: {
+        type: new GraphQLNonNull(GraphQLAccountCollection),
+        description: 'Returns a list of accounts hosted by this host',
+        args: {
+          ...getCollectionArgs({ limit: 100, offset: 0 }),
+          accountType: { type: new GraphQLList(GraphQLAccountType) },
+          isApproved: {
+            type: GraphQLBoolean,
+            description: 'Filter on (un)approved collectives',
+            defaultValue: true,
+          },
+          isFrozen: {
+            type: GraphQLBoolean,
+            description: 'Filter on frozen accounts',
+            defaultValue: false,
+          },
+          isUnhosted: {
+            type: GraphQLBoolean,
+            description: 'Filter on unhosted accounts',
+            defaultValue: false,
+          },
+          hostFeesStructure: {
+            type: GraphQLHostFeeStructure,
+            description: 'Filters on the Host fees structure applied to this account',
+          },
+          searchTerm: {
+            type: GraphQLString,
+            description:
+              'A term to search membership. Searches in collective tags, name, slug, members description and role.',
+          },
+          orderBy: {
+            type: GraphQLOrderByInput,
+            description: 'Order of the results',
+          },
+        },
+        async resolve(host, args, req) {
+          const isAdmin = req.remoteUser?.isAdmin(host.id);
+          if (!isAdmin) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see its hosted accounts');
+          }
```

Comment:
Could this be public? Seems useful to use this query also on the public profile pages of a host to display hosted collectives instead of using `memberOf`